scripts/docker/build.sh

```diff
@@ -33,6 +33,7 @@ Build specified docker image targets. Available targets:
 - astrobee_base
 - astrobee
 - test_astrobee
+- astrobee_quick
```

Comment:
I think it would be helpful to document somewhere what your recommended workflow looks like for using the `astrobee_quick` target.
- Do you just always use it by default and it makes things run faster? Or do you have to build `astrobee` at least once before it works to build `astrobee_quick`?
- Is there ever a downside to always using `astrobee_quick`? (Like: "If you think your build is breaking because the cache is corrupted somehow, here's how to clear it.")